From 1af4a2c0eed60017741611d15b22bcdaffc97453 Mon Sep 17 00:00:00 2001
Message-Id: <1af4a2c0eed60017741611d15b22bcdaffc97453.1383564115.git.minovotn@redhat.com>
In-Reply-To: <5575e0aec51f40ebec46e98ec085cda053283aba.1383564115.git.minovotn@redhat.com>
References: <5575e0aec51f40ebec46e98ec085cda053283aba.1383564115.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Fri, 27 Sep 2013 13:31:16 +0200
Subject: [PATCH 06/14] doc: Drop ref to Bochs from -no-fd-bootchk
 documentation

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1380288680-26645-7-git-send-email-armbru@redhat.com>
Patchwork-id: 54562
O-Subject: [PATCH 7.0 qemu-kvm 06/10] doc: Drop ref to Bochs from -no-fd-bootchk documentation
Bugzilla: 997817
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

Manual page and qemu-doc on talk about "Bochs BIOS".  We use SeaBIOS,
and it implements the feature.  Replace by just "BIOS", and drop the
TODO line wondering about the Bochs reference.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Reviewed-by: Anthony Liguori <aliguori@us.ibm.com>
Message-id: 1371208516-7857-7-git-send-email-armbru@redhat.com
Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 4eda32f588086b6cd0ec2be6a7a6c131f8c2b427)
---
 qemu-options.hx | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-options.hx | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index 202a2cf..3c60140 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -1268,9 +1268,8 @@ DEF("no-fd-bootchk", 0, QEMU_OPTION_no_fd_bootchk,
 STEXI
 @item -no-fd-bootchk
 @findex -no-fd-bootchk
-Disable boot signature checking for floppy disks in Bochs BIOS. It may
+Disable boot signature checking for floppy disks in BIOS. May
 be needed to boot from old floppy disks.
-TODO: check reference to Bochs BIOS.
 ETEXI
 
 DEF("no-acpi", 0, QEMU_OPTION_no_acpi,
-- 
1.7.11.7

